### ISSUE TITLE
When I access R shiny with IE, plotly is not displayed.

### ISSUE THREAD

**Issue by ToshihiroIguchi** [ropensci/plotly#1605]
When I start shiny by specifying host and port with runApp on Windows10 Home 32bit and access with IE11, there is a problem that the plotly graph is not displayed for the first time.
The version used was R 3.6.0, shiny 1.3.2, plotly 4.9.0.

However, it was displayed when IE was reloaded.

Also, if you start shiny after changing host or port with runApp, the plotly graph will not be displayed for the first time.

Below is the code used to start ui.R and server.R.

ui.R
```r
library(shiny)

shinyUI(fluidPage(
  titlePanel("test"),
  plotlyOutput("plt")
))
```

server.R
```r
library(shiny)
library(plotly)

shinyServer(function(input, output) {
  output$plt <- renderPlotly({plot_ly()})
})
```

It started with the following code.
```r
ipconfig.dat <- system("ipconfig", intern = TRUE)
ipv4.dat <- ipconfig.dat[grep("IPv4", ipconfig.dat)][1]
ip <- gsub(".*? ([[:digit:]])", "\\1", ipv4.dat)
shiny::runApp(launch.browser = FALSE, port = 8080, host = ip)
```

In addition, the image file of the screenshot when accessing the first time and the third time with IE11 is attached.

The same problem occurred when launching shiny on Windows 10 Pro 64bit.
The same problem occurred when accessing with Edge.
However, this problem did not occur when accessed via Chrome.

Is it possible to display plotly graphs from the first time even when accessing with IE11?

![IE_1st](https://user-images.githubusercontent.com/20187797/63639046-5566ba80-c6ca-11e9-8e39-9227e1e01480.png)
![IE_3rd](https://user-images.githubusercontent.com/20187797/63639047-57307e00-c6ca-11e9-8a7f-c6e5a32699cb.png)

**Comment by cpsievert** [comment]
Interesting, thanks, I was able reproduce. When you right-click -> "Inspect" -> Console, do you see an error similar to this one?

<img width="1018" alt="Screen Shot 2019-08-28 at 4 43 40 PM" src="https://user-images.githubusercontent.com/1365941/63894803-2c3c7780-c9b3-11e9-9a07-41ad54aeb3c4.png">

@wch @jcheng5 I'm pretty sure this server error will occur for any htmlwidget that uses the `dependency` argument of `htmlwidgets::saveWidget()`.

**Comment by cpsievert** [comment]
For posterity:

```r
- Session info ---------------------------------------------
 setting  value                       
 version  R version 3.5.2 (2018-12-20)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/Los_Angeles         
 date     2019-08-28                  

- Packages -------------------------------------------------
 package     * version    date       lib
 assertthat    0.2.1      2019-03-21 [1]
 backports     1.1.4      2019-04-10 [1]
 callr         3.3.1      2019-07-18 [1]
 cli           1.1.0      2019-03-19 [1]
 colorspace    1.4-1      2019-03-18 [1]
 crayon        1.3.4      2017-09-16 [1]
 crosstalk     1.0.0      2016-12-21 [1]
 data.table    1.12.2     2019-04-07 [1]
 desc          1.2.0      2018-05-01 [1]
 devtools      2.1.0      2019-07-06 [1]
 digest        0.6.20     2019-07-04 [1]
 dplyr         0.8.3      2019-07-04 [1]
 fs            1.3.1      2019-05-06 [1]
 ggplot2     * 3.1.0      2018-10-25 [1]
 glue          1.3.1      2019-03-12 [1]
 gtable        0.3.0      2019-03-25 [1]
 htmltools     0.3.6      2017-04-28 [1]
 htmlwidgets   1.3        2018-09-30 [1]
 httpuv        1.5.1      2019-04-05 [1]
 httr          1.4.1      2019-08-05 [1]
 jsonlite      1.6        2018-12-07 [1]
 later         0.8.0      2019-02-11 [1]
 lazyeval      0.2.2      2019-03-15 [1]
 magrittr      1.5        2014-11-22 [1]
 memoise       1.1.0      2017-04-21 [1]
 mime          0.7        2019-06-11 [1]
 munsell       0.5.0      2018-06-12 [1]
 packrat       0.5.0      2018-11-14 [1]
 pillar        1.4.2      2019-06-29 [1]
 pkgbuild      1.0.5      2019-08-26 [1]
 pkgconfig     2.0.2      2018-08-16 [1]
 pkgload       1.0.2      2018-10-29 [1]
 plotly      * 4.8.0      2018-07-20 [1]
 plyr          1.8.4      2016-06-08 [1]
 prettyunits   1.0.2      2015-07-13 [1]
 processx      3.4.1      2019-07-18 [1]
 promises      1.0.1      2018-04-13 [1]
 ps            1.3.0      2018-12-21 [1]
 purrr         0.3.2      2019-03-15 [1]
 R6            2.4.0      2019-02-14 [1]
 Rcpp          1.0.2      2019-07-25 [1]
 remotes       2.1.0      2019-06-24 [1]
 rlang         0.4.0      2019-06-25 [1]
 rprojroot     1.3-2      2018-01-03 [1]
 rstudioapi    0.10       2019-03-19 [1]
 scales        1.0.0.9000 2019-02-28 [1]
 sessioninfo   1.1.1      2018-11-05 [1]
 shiny       * 1.3.2      2019-04-22 [1]
 testthat      2.2.1      2019-07-25 [1]
 tibble        2.1.3      2019-06-06 [1]
 tidyr         0.8.3      2019-03-01 [1]
 tidyselect    0.2.5      2018-10-11 [1]
 usethis       1.5.1      2019-07-04 [1]
 viridisLite   0.3.0      2018-02-01 [1]
 withr         2.1.2      2018-03-15 [1]
 xtable        1.8-4      2019-04-21 [1]
 yaml          2.2.0      2018-07-25 [1]
 source                       
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.3)               
 Github (r-lib/scales@c374014)
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.2)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.1)               
 CRAN (R 3.5.3)               
 CRAN (R 3.5.1)               

[1] C:/Users/Carson Sievert/Documents/R/win-library/3.5
[2] C:/Program Files/R/R-3.5.2/library
```

**Comment by ToshihiroIguchi** [comment]
> Interesting, thanks, I was able reproduce. When you right-click -> "Inspect" -> Console, do you see an error similar to this one?

The error of the same content was confirmed in the console of Edge.
The screenshot image I attached has a message written in Japanese, but it is almost the same as the English message you attached.
The SCRIPT1004 error was also displayed.

![edge_console](https://user-images.githubusercontent.com/20187797/63940819-3b035700-caa5-11e9-89b8-37926a42012d.png)


Nothing was displayed on the IE11 console.
This also attaches an image.

![console](https://user-images.githubusercontent.com/20187797/63940846-48204600-caa5-11e9-86a8-4491e5d8639e.png)


In addition, the graph is displayed if the port number is the same as the previous time, so I changed it to another port number.
